### PR TITLE
Fix horizontal navigation menu example

### PR DESCRIPTION
**Pull request by maspitz**
#### Summary

Removes the spaces and line breaks between the &lt;li&gt; elements
in the horizontal nav menu example.

#### Motivation

As noted in the the tutorial's text, there must be no spaces or line spaces
between the &lt;li&gt; elements.

Otherwise, the fifth item ("Finland") ends up being placed
on a line by itself, contrary to the intended nav menu design.

#### Supporting details

https://css-tricks.com/fighting-the-space-between-inline-block-elements/

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/css/styling_text/styling_links/index.md [hunk 1]
```diff
@@ -387,11 +387,7 @@ First, some HTML:
 
 ```html
 <ul>
-  <li><a href="#">Home</a></li>
-  <li><a href="#">Pizza</a></li>
-  <li><a href="#">Music</a></li>
-  <li><a href="#">Wombats</a></li>
-  <li><a href="#">Finland</a></li>
+  <li><a href="#">Home</a></li><li><a href="#">Pizza</a></li><li><a href="#">Music</a></li><li><a href="#">Wombats</a></li><li><a href="#">Finland</a></li>
```
Comment (OnkarRuikar) [inline files/en-us/learn/css/styling_text/styling_links/index.md:390]: I think the comment fix would be more suitable/presentable for the demo. And it'll also shine more light on the issue. 
```suggestion
  <li><a href="#">Home</a></li><!--
  --><li><a href="#">Pizza</a></li><!--
  --><li><a href="#">Music</a></li><!--
  --><li><a href="#">Wombats</a></li><!--
  --><li><a href="#">Finland</a></li>
```

File: files/en-us/learn/css/styling_text/styling_links/index.md [hunk 2]
```diff
@@ -387,7 +387,11 @@ First, some HTML:
 
 ```html
 <ul>
-  <li><a href="#">Home</a></li><li><a href="#">Pizza</a></li><li><a href="#">Music</a></li><li><a href="#">Wombats</a></li><li><a href="#">Finland</a></li>
+  <li><a href="#">Home</a></li><!--
+  --><li><a href="#">Pizza</a></li><!--
+  --><li><a href="#">Music</a></li><!--
+  --><li><a href="#">Wombats</a></li><!--
+  --><li><a href="#">Finland</a></li>
```
Comment (teoli2003) [inline files/en-us/learn/css/styling_text/styling_links/index.md:394]: I think this is the more legible.

```suggestion
  <li><a href="#">Home</a></li>
  <li><a href="#">Pizza</a></li>
  <li><a href="#">Music</a></li>
  <li><a href="#">Wombats</a></li>
  <li><a href="#">Finland</a></li>
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/css/styling_text/styling_links/index.md (hunk 2 above)
Comment (teoli2003) [reply files/en-us/learn/css/styling_text/styling_links/index.md:394]: Oh yes, you are right, I remember. I complained about it once. So I think we should keep it horizontal then.

Comment (maspitz) [reply files/en-us/learn/css/styling_text/styling_links/index.md:394]: Just so I understand clearly, by keeping it horizontal, do you mean, you prefer the method of the single line `<li><a href="#">Home</a></li><li><a href="#">Pizza</a></li><li><a href="#">Music</a></li><li><a href="#">Wombats</a></li><li><a href="#">Finland</a></li>` without comment-protected newlines?  Thanks.

Comment (teoli2003) [reply files/en-us/learn/css/styling_text/styling_links/index.md:394]: Yes, I do. If we add comment-protected newlines, we should explain why and I think this is out-of-scope for this page.

Comment (maspitz) [reply files/en-us/learn/css/styling_text/styling_links/index.md:394]: Ok, it is the simplest way, and the example is already relatively complicated.

```suggestion
  <li><a href="#">Home</a></li><li><a href="#">Pizza</a></li><li><a href="#">Music</a></li><li><a href="#">Wombats</a></li><li><a href="#">Finland</a></li>
```

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Pull request merged**